File: fix_campaigns.py
with open('backend-repo/src/modules/campaigns/campaigns.service.ts', 'r') as f:
    content = f.read()

old = 'prisma.\(async (tx) => {'
new = 'prisma.$transaction(async (tx) => {'
print('old repr:', repr(old))
print('found:', old in content)
content = content.replace(old, new)

with open('backend-repo/src/modules/campaigns/campaigns.service.ts', 'w') as f:
    f.write(content)

File: fix_campaigns.js
const fs = require('fs');  
const file = 'C:/Users/LENOVO/Desktop/veakay-handoff/backend-repo/src/modules/campaigns/campaigns.service.spec.ts';  
let content = fs.readFileSync(file, 'utf8');  
content = content.replace('service = new CampaignsService(\\n      prisma,\\n      mediaAssetsService,\\n      adminAuditLogService,\\n      verifiedBadgesService,\\n    );', 'const mockNotificationsService = { create: jest.fn().mockResolvedValue({}) };\\n    service = new CampaignsService(\\n      prisma,\\n      mediaAssetsService,\\n      adminAuditLogService,\\n      verifiedBadgesService,\\n      mockNotificationsService as any,\\n    );');  
fs.writeFileSync(file, content);  
console.log('Fixed campaigns.service.spec.ts'); 
